sources/packages/api/src/route-controllers/institution/models/institution-user-auth.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsIn, IsNotEmpty, IsOptional } from "class-validator";
+import { InstitutionUserType, InstitutionUserRole } from "../../../types";
+
+export class InstitutionUserAuthDto {
+  @IsNotEmpty()
+  guid: string;
```

Comment:
Please consider making explicit that this is the user guid.

Reply:
The domain representation of the prop will be InstitutionUserAuthDto.guid, that implicitly tagged as user_guid